docs/hugo/content/introduction/installing-from-yaml.md

```diff
@@ -11,7 +11,7 @@ title: "Installation: From YAML"
 
 1. Install [the latest **v2+** release](https://github.com/Azure/azure-service-operator/releases) of Azure Service Operator.
    ```bash
-   kubectl apply --server-side=true -f https://github.com/Azure/azure-service-operator/releases/download/v2.0.0-beta.0/azureserviceoperator_v2.0.0-beta.0.yaml
+   kubectl apply --server-side=true -f https://github.com/Azure/azure-service-operator/releases/download/v2.0.0-beta.3/azureserviceoperator_v2.0.0-beta.3.yaml
```

Comment:
can we have something like `latest` here? so that we dont change this on every release.

Reply:
Technically we could, but I believe that we decided we didn't want to document using latest because it's a bit of a bad practice. Using it in user scripts or anything like that might result in getting 2 different versions of ASO without the user actually opting in to doing so.

I think it's OK to hardcode the version here. Even if we don't update it, we tell them to install the latest v2 release (and give a link). This isn't the preferred installation either, that's Helm.

Between those two mitigations, I don't think people will be getting too confused by this even if it doesn't get updated every time.